vaadinfrontend/frontend/themes/datamanager/components/card.css

```diff
@@ -1,5 +1,9 @@
 /* All custom css for components which extend the card component go in here! */
 
+.title {
+  font-weight: bold;
+}
```

Comment:
This is dangerous since it title is a common classname and this will overwrite it in all our css components.  
I'd suggest something like:
`.sample-origin-title `

Reply:
Since a style guide is missing, i assumed titles are always bold, no matter where they are used. Surely, this can be made more specific. However, if we want to change the looks of card titles, we need to touch a lot of css properties, if we go that way.

Reply:
`.card .title {...}`  => you do not overwrite `.title`
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors